pkg/ddl/backfilling_merge_sort.go

```diff
@@ -88,16 +88,15 @@ func (m *mergeSortExecutor) RunSubtask(ctx context.Context, subtask *proto.Subta
 	}
 
 	prefix := path.Join(strconv.Itoa(int(m.jobID)), strconv.Itoa(int(subtask.ID)))
-	partSize, err := getMergeSortPartSize(m.avgRowSize, subtask.Concurrency, m.idxNum)
```

Comment:
m.avgRowSize is also not used any more

Reply:
23850e41930eb5a95e4f61f3309e4488a3cfc103